### PR TITLE
chore: set base retry after value to 5s instead of the default of 1s

### PR DESCRIPTION
**Pull request by dgellow**

### CODE REVIEW

File: src/github.ts [hunk 1]
```diff
@@ -328,6 +327,8 @@ export class GitHub {
           error: logger.error,
         },
         retry: {
+          retries: options.retries ?? 0,
+          retryAfter: 3, // 3 seconds
```
Comment (dgellow) [inline src/github.ts:331]: This is the number of second between retries.

File: src/github.ts [hunk 2]
```diff
@@ -336,6 +337,8 @@ export class GitHub {
         },
         throttle: {
           enabled: throttlingRetries > 0,
+          retryAfterBaseValue: 3000, // 3 seconds
```
Comment (dgellow) [inline src/github.ts:340]: This is the number that will be multiplied with headers `retry-after` or `x-ratelimit-reset`. So we are multiplying by 3 what the server tells us.

File: src/github.ts [hunk 3]
```diff
@@ -328,6 +327,8 @@ export class GitHub {
           error: logger.error,
         },
         retry: {
+          retries: options.retries ?? 0,
+          retryAfter: 3, // 3 seconds
           doNotRetry: [
             '403', // Used by GitHub when throttling
```
Comment (marcel-stainless) [inline src/github.ts:333]: Ooh, that's interesting. 403's where the error we were seeing. Why github abuses this error code is beyond me, but oh well. So we do need to retry on 403 (but slowly)

Comment (dgellow) [reply src/github.ts:333]: It's a bit weird, but we are using two octokit plugins (that's recommended by their documentation). One to retry general errors, and one to handle throttling. I added 403 to the list on "do not retry" just to be sure the `retry` plugin doesn't handle retry for cases where the `throttling` plugin should.

**Review by marcel-stainless — APPROVED** [review]

**Comment by marcel-stainless** [comment]
We should probably also increase the number or retries. Failing some of these cause intricate issues

**Pull request merged**